include/Sugar_Smarty.php

```diff
@@ -157,16 +157,16 @@ public function fetch($resource_name, $cache_id = null, $compile_id = null, $dis
         $this->assign('MOD', $mod_strings);
         $this->assign('APP_CONFIG', $sugar_config);
 
-        $state = new SuiteCRM\StateSaver();
-        $state->pushErrorLevel('sugar_smarty_errors');
```

Comment:
maybe more proper solution is adding a feature toggling into push/popErrorLevel function here: (for e.g)
https://github.com/salesagility/SuiteCRM/blob/master/include/StateSaver.php#L302
like this:
```
 public function pushErrorLevel($key = 'level', $namespace = 'error_reporting', $doLogging = true)
    {
        if ($doLogging) {
            LoggerManager::getLogger()->warn('Saving error level. Try to remove the error_reporting() function from your code.');
        }
        $level = error_reporting();
        $this->push($level, $key, $namespace);
    }
```
and just simple call with the extra parameter like this:
`$state->pushErrorLevel('sugar_smarty_errors', 'smarty_error_reporting', false);`
..and do same thing at `popErrorLevel`.